fdbserver/RestoreLoader.actor.cpp

```diff
@@ -945,6 +949,9 @@ ACTOR Future<Void> handleFinishVersionBatchRequest(RestoreVersionBatchRequest re
 	wait(self->finishedBatch.whenAtLeast(req.batchIndex - 1));
 	if (self->finishedBatch.get() == req.batchIndex - 1) {
 		self->finishedBatch.set(req.batchIndex);
+		// Clean up batchData
+		self->batch.erase(req.batchIndex);
+		self->status.erase(req.batchIndex);
```

Comment:
Might be better to refactor this into a member function, e.g., `self->finishBatch(req.batchIndex);`.

Reply:
Yeah, I thought about that. Right now, it only does these two operations.
In the initVersionBatch case, I have the member function to do that. I prefer to leave it as it is for now.